upgrading to the latest revision of panther (10.3.9) solved this, i can now successfully use my external drive. 
thanks for input. 
